### PR TITLE
remake Zipcode field

### PR DESCRIPTION
**Pull request by GoodDayForSurf**

### CODE REVIEW

File: packages/vue/src/pages/crm-contact-list/crm-contact-list.vue [hunk 1]
```diff
@@ -232,6 +232,7 @@ const dataSource = new DataSource({
 });
 
 const rowClick = (e: RowClickEvent) => {
+  console.log('rowClick', e.data.id);
```
Comment (Krijovnick) [inline packages/vue/src/pages/crm-contact-list/crm-contact-list.vue:235]: remove console.log

File: packages/angular/src/app/shared/types/contact.ts [hunk 2]
```diff
@@ -50,7 +50,7 @@ export const newContact: Contact = {
   email: '',
   image: '',
   address: '',
-  zipCode: '',
+  zipCode: null,
```
Comment (Krijovnick) [inline packages/angular/src/app/shared/types/contact.ts:53]: It is weird: zipCode is null in data and it is number in types at the same time and it works without errors

File: packages/react/src/components/contact-form/ContactFormDetails.tsx [hunk 3]
```diff
@@ -6,14 +6,15 @@ import From, { Item as ItemForm, GroupItem, ColCountByScreen } from 'devextreme-
...
-import { PatternRule, EmailRule } from 'devextreme-react/validator';
+import Validator, { PatternRule, EmailRule } from 'devextreme-react/validator';
 
 import { FormPhoto } from '../form-photo/FormPhoto';
 import { ContactStatus } from '../contact-status/ContactStatus';
 import { FormTextbox } from '../form-textbox/FormTextbox';
 
 import { Contact } from '../../shared/types/crm-contact';
 import { CONTACT_STATUS_LIST } from '../../shared/constants';
+import { NumberBox } from 'devextreme-react';
```
Comment (Krijovnick) [inline packages/react/src/components/contact-form/ContactFormDetails.tsx:17]: After build in console appears errors with this record. There is should be:
import NumberBox from 'devextreme-react/number-box;
And move it to top, after `Validator`, to make it more consistently

File: packages/angular/src/app/shared/types/contact.ts (hunk 2 above)
Comment (GoodDayForSurf) [reply packages/angular/src/app/shared/types/contact.ts]: i've remake type

**Review by Krijovnick — DISMISSED** [review]

File: packages/react/src/components/contact-form/ContactFormDetails.tsx [hunk 4]
```diff
@@ -146,17 +147,22 @@ export const ContactFromDetails = ({ data, editing }: { data: Contact, editing:
         </ItemForm>
 
         <ItemForm>
-          <FormTextbox
+          <NumberBox
```
Comment (dxvladislavvolkov) [inline packages/react/src/components/contact-form/ContactFormDetails.tsx:150]: I see that you use number-box here but we also use FormTextbox could we replace it with text-box from devextreme? Also, it is an incorrect name. Should be FormTextBox

File: packages/vue/src/pages/crm-contact-details/components/contact-form.vue [hunk 5]
```diff
@@ -67,11 +67,9 @@
                 v-model="contactData.status"
                 :items="contactStatusList"
                 :read-only="!isEditing"
-                :element-attr="{class: 'form-editor'}"
-                :input-attr="{class: 'form-editor-input'}"
-                styling-mode="filled"
                 field-template="field"
                 item-template="item"
+                v-bind="formEditorSelectBoxProps"
```
Comment (dxvladislavvolkov) [inline packages/vue/src/pages/crm-contact-details/components/contact-form.vue:72]: formEditorSelectBoxProps -> formSelectBoxEditorProps

**Review by dxvladislavvolkov — APPROVED** [review]

**Review by Krijovnick — APPROVED** [review]

**Pull request merged**